cognite/neat/rules/transformers/_converters.py

```diff
@@ -38,34 +40,49 @@
 )
 from cognite.neat.rules.models.information import InformationClass, InformationMetadata, InformationProperty
 
-from ._base import RulesTransformer
+from ._base import JustRules, OutRules, RulesTransformer
 
+T_VerifiedInRules = TypeVar("T_VerifiedInRules", bound=VerifiedRules)
+T_VerifiedOutRules = TypeVar("T_VerifiedOutRules", bound=VerifiedRules)
 
-class InformationToDMS(RulesTransformer[InformationRules, DMSRules]):
+
+class ConversionTransformer(RulesTransformer[T_VerifiedInRules, T_VerifiedOutRules], ABC):
+    """Base class for all conversion transformers."""
+
+    def transform(self, rules: T_VerifiedInRules | OutRules[T_VerifiedInRules]) -> JustRules[T_VerifiedOutRules]:
+        out = self._transform(self._to_rules(rules))
+        return JustRules(out)
```

Comment:
Note to myself: conversion must always succeeded , so `JustRules` is our return type

Reply:
That is absolutely up for discussion.